modules/@apostrophecms/schema/index.js

```diff
@@ -599,7 +599,7 @@ module.exports = {
       // set error class names, etc. If the error is not a string, it is a
       // database error etc. and should not be displayed in the browser directly.
 
-      async convert(req, schema, data, destination) {
+      async convert(req, schema, data, destination, options) {
```

Comment:
You mean for readability?
Because for me the safer is to handle the default value in the field convert itself.
Since convert methods are called from a lot  of places without options most of the time it feels safer and less verbose to have one true default location where the option is actually used. WDYT?